src/app/views/profile/trophies/overview/overview.ts

```diff
@@ -20,6 +21,8 @@ type TrophyEntry = {
 	trophies: UserBaseTrophy[];
 };
 
+const PAGE_SIZE = 12;
```

Comment:
rename to `PAGE_SIZE_PER_TROPHY_TYPE` and add a comment saying 12 is chosen arbitrarily (and not chosen to satisfy any design layout constraints).

Reply:
actually, this should be returned from the backend, otherwise you might not allow loading more when you should be able to. could tweak without having to push frontend.